models/info.go

```diff
@@ -0,0 +1,61 @@
+package models
+
+import (
+	"encoding/json"
+)
+
+// Info maps AsyncAPI info object
+type Info struct {
+	ExtensionProps ExtensionProps `json:"-"`
+	Title          string         `json:"title,omitempty"`
+	Description    string         `json:"description,omitempty"`
+	TermsOfService string         `json:"termsOfService,omitempty"`
+	Contact        *Contact       `json:"contact,omitempty"`
+	License        *License       `json:"license,omitempty"`
+	Version        string         `json:"version,omitempty"`
+}
+
+// Unmarshal unmarshals JSON
+func (value *Info) Unmarshal(data []byte) error {
```

Comment:
Is there a way we can create an interface for all models to implement Unmarshal and Marshal. I'm seen now these two methods all over the place doing the exact thing

Reply:
Yes, would be great to avoid repetition.